core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/DataRowApi.kt

```diff
@@ -166,12 +169,16 @@ public fun AnyRow.columnNames(): List<String> = df().columnNames()
 
 public fun AnyRow.columnTypes(): List<KType> = df().columnTypes()
 
+@CandidateForRemoval
```

Comment:
I think these ones should just be documented and better tested. It's in the same family of functions as `relative` which is powerful in the `add` DSL. And to populate a column with values using, for instance `getRow(0).someCol` as part of the calculation seems quite critical. Again, we just need more examples. Though, marking them as CandidateForRemoval is a good way for us to start looking into it again later :)